test/util/testnode/full_node.go

```diff
@@ -147,13 +147,15 @@ func DefaultTendermintConfig() *config.Config {
 	// during tests.
 	tmCfg.Consensus.TargetHeightDuration = 300 * time.Millisecond
 	tmCfg.Consensus.TimeoutPropose = 200 * time.Millisecond
-	tmCfg.Mempool.MaxTxBytes = 22020096 // 21MB
+	tmCfg.Mempool.MaxTxBytes = 220020096 // 210MB
+	tmCfg.RPC.MaxBodyBytes = 220020096   // 210MB
```

Comment:
[blocking]

```suggestion
	tmCfg.Mempool.MaxTxBytes = 220020096 // 21 MB
	tmCfg.RPC.MaxBodyBytes = 220020096   // 21 MB
```

Ref: https://www.flightpedia.org/convert/21-megabytes-to-bytes.html

Reply:
wait 220_020_096 is very close to 210MB no?

Reply:
hmm I'm confused because the comment was updated but the actual number wasn't. Should this be 21 MB or 210MB. Either way, proposal to use underscores in the number to make it easier to read:


21 MB
```go
	tmCfg.Mempool.MaxTxBytes = 22_020_096 // 21 MB
	tmCfg.RPC.MaxBodyBytes = 22_020_096   // 21 MB
```

210 MB

```go
	tmCfg.Mempool.MaxTxBytes = 220_200_960 // 210 MB
	tmCfg.RPC.MaxBodyBytes = 220_200_960   // 210 MB
```

Reply:
https://github.com/celestiaorg/celestia-app/pull/1772/commits/0467644818d6e02ab2d7cc45be9139009b2d91c2